README.md

```diff
@@ -85,6 +85,38 @@ A huge thank you for all of the backers / sponsors!
 
 This plugin integrates [Vue](https://vuejs.org/) and [NativeScript](https://www.nativescript.org/), allowing you to build cross-platform iOS and Android apps using Vue.
 
+
+# console.log
```

Comment:
Let's move this to a separate md file - `CONSOLE_LOG_OVERRIDE.md` and we will link to it from the changelog's `BREAKING CHANGE` section.